techniques/applications/zypperPackageManagerRepositories/1.0/zypper-repositories-management.st

```diff
@@ -75,6 +75,14 @@ bundle agent zypper_repositories_management
     SuSE::
       "repos_${zypper_index}" usebundle => rudder_common_reports_generic("Zypper Repositories Management", "zypper_repo_${zypper_index}", "${zypper_uuid[${zypper_index}]}", "Repositories", "${zypper_name[${zypper_index}]}", "Repository ${zypper_name[${zypper_index}]}");
 
+      # Clear package caches
+      "remove_caches" usebundle => file_remove("/var/rudder/cfengine-community/state/software_packages.csv"),
```

Comment:
Doesn't this contain the installed package list ?
If so, why would you need to update it after a repository change ?